src/FunctionApps/python/tests/IntakePipeline/transform_test.py

```diff
@@ -86,7 +86,7 @@ def test_transform_bundle(patched_geocode, combined_bundle):
         ],
     }
 
-    transform_bundle(mock.Mock(), combined_bundle)
+    transform_bundle(mock.Mock(), combined_bundle, add_std_extension=False)
```

Comment:
$0.02: isn't `False` the default?

Reply:
Luckily this won't be an issue anymore with us killing off the configurability 💪